### ISSUE TITLE
Assistant for push messages in GUI

### ISSUE THREAD

**Issue by sebw** [gotify/server#166]
I like how Simplepush presents a couple of different ways to generate push messages on their website:

https://simplepush.io/#libraries

How about having a similar feature directly in the server GUI? That would include the bits documented here https://gotify.net/docs/pushmsg

**Comment by eternal-flame-AD** [comment]
We already have the API documentation included in the binary. Doing so would require baking documentations into the binary as well.

@jmattheis What do you think about this? I think a link to the official documentation in the WebUI should be enough.

**Comment by sebw** [comment]
Indeed, let's keep the code base light.
A simple link taking to examples would be perfect!

**Comment by jmattheis** [comment]
Yeah, we can include a link to the documentation. How about adding the following links under the list of applications in the left navigation
* Documentation
* Source Code
* Issues

**Comment by eternal-flame-AD** [comment]
I would prefer a page footer that include links to the documentation. The left navigation should be reserved only for selecting applications.

**Comment by pm-pm** [comment]
Don´t add links to "**admin** dokumentation" to "**user** webpages" ;-)
But a inconspicuous link of course is helpful.

**Comment by sebw** [comment]
I actually discovered https://gotify.net/docs/more-pushmsg just now. That's exactly the kind of example I was looking for when opening this case.

Can we merge https://gotify.net/docs/pushmsg and https://gotify.net/docs/more-pushmsg ? I can take care of it.

**Comment by jmattheis** [comment]
@sebw I did the separation because there are many programming languages for which examples could be added. I'd say a link (which exists) to the more pushmsg page is enough. Maybe we can highlight this link a little more.

**Comment by sebw** [comment]
@jmattheis yup, makes sense. Renaming to Library and putting it right after pushmsg would probably be a bit clearer?

**Comment by jmattheis** [comment]
Renaming is okay, I don't think it fits into getting started, as it is a more advanced usage of the gotify/server api.
